### ISSUE TITLE
Existing proposition shall be displayed in announcement

### ISSUE THREAD

**Issue by YannickB** [xpansa/community-web#3]
Actually, when we go in an announcement the existing propositions aren't displayed.

**Comment by veryberry** [comment]
Actually, propositions are visible according to ir.rule entities (there are several ones defined for marketplace.proposition and account.wallet.transaction models). Now 'open' state and 'already_published' are written to proposition at the moment of creation. Mb there is some issue in proposition-transaction state sync workflow..

**Comment by YannickB** [comment]
Maybe... At the very least, the proposition created by the demo data are not displayed.

**Comment by veryberry** [comment]
Yannick,
I've checked in DB, transactions related to demo propositions have state 'draft' and transactions have its own ir.rule entities, that is cause why proposition aren't displayed in this case. Should I do any actions from my side to correct this situation?

**Comment by YannickB** [comment]
Ok I realised during my test with Julien, In fact I was tricked by the fact I went directly redirected in edit mode, and propositions aren't displayed in edit mode :). That's why I though proposition weren't displayed.

Julien made an error not asking for a simple edit/view mode for announcement, like you did on member page, and so the ergonomy is quite awful on this part. I opened a ticket about it #14 , for this ticket I think we can close it.
